pkg/distributors/test/MultiRewards.test.ts

```diff
@@ -33,13 +32,29 @@ describe('Staking contract', () => {
   sharedBeforeEach('set up asset manager', async () => {
     const { contracts } = await setup();
 
+    authorizer = contracts.authorizer;
     pool = contracts.pool;
     vault = contracts.vault;
     stakingContract = contracts.stakingContract;
     rewardToken = contracts.rewardTokens.DAI;
     rewardTokens = contracts.rewardTokens;
   });
 
+  describe('authorizer', () => {
+    it('uses the authorizer of the vault', async () => {
+      expect(await stakingContract.getAuthorizer()).to.equal(authorizer.address);
+    });
+
+    it('tracks authorizer changes in the vault', async () => {
+      const action = await actionId(vault, 'setAuthorizer');
```

Comment:
```suggestion
      const action = actionId(vault, 'setAuthorizer');
```

This function is synchronous so we don't need an await.